### PR TITLE
Update Rust and frozen flame

### PR DESCRIPTION
**Pull request by Wapriaily**
# Description

Update Rust query port and app port
Fix frozen flame some text errors

## Checklist for all submissions

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/parkervcp/eggs/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you tested and reviewed your changes with confidence that everything works?
* [x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

### CODE REVIEW

File: game_eggs/steamcmd_servers/rust/rust_autowipe/egg-rust-autowipe.json [hunk 1]
```diff
@@ -4,9 +4,9 @@
         "version": "PTDL_v2",
         "update_url": null
     },
-    "exported_at": "2022-11-26T11:35:12+01:00",
+    "exported_at": "2023-02-03T13:00:17+08:00",
     "name": "Rust Autowipe",
-    "author": "support@pterodactyl.io",
+    "author": "kano@wapriaily.com",
```
Comment (QuintenQVD0) [inline game_eggs/steamcmd_servers/rust/rust_autowipe/egg-rust-autowipe.json:9]: Swapping author is not done

File: game_eggs/steamcmd_servers/frozen_flame/egg-frozen-flame.json [hunk 2]
```diff
@@ -50,7 +50,7 @@
         },
         {
             "name": "Server Name",
-            "description": "The server nane",
+            "description": "The server name",
```
Comment (QuintenQVD0) [inline game_eggs/steamcmd_servers/frozen_flame/egg-frozen-flame.json:53]: This was not exported from a panel

**Review by QuintenQVD0 — CHANGES_REQUESTED** [review]

**Comment by QuintenQVD0** [comment]
Aslo fill in the pr checklist

**Comment by Wapriaily** [comment]
Changed

File: game_eggs/steamcmd_servers/frozen_flame/egg-frozen-flame.json [hunk 3]
```diff
@@ -4,9 +4,9 @@
         "version": "PTDL_v2",
         "update_url": null
     },
-    "exported_at": "2022-11-19T18:35:39+01:00",
+    "exported_at": "2023-02-03T14:32:26+08:00",
     "name": "Frozen Flame",
-    "author": "theblitzbat@gmail.com",
+    "author": "parker@parkervcp.com",
```
Comment (parkervcp) [inline game_eggs/steamcmd_servers/frozen_flame/egg-frozen-flame.json:9]: why did you change the author here?

**Review by parkervcp — CHANGES_REQUESTED** [review]

File: game_eggs/steamcmd_servers/frozen_flame/egg-frozen-flame.json (hunk 3 above)
Comment (Wapriaily) [reply game_eggs/steamcmd_servers/frozen_flame/egg-frozen-flame.json:9]: Sorry, I export that's it.

**Review by Software-Noob — APPROVED** [review]
It'd be ideal in the future to have one PR for each egg. LGTM :+1:

**Review by parkervcp — APPROVED** [review]

**Comment by Wapriaily** [comment]
Can it be merged? @QuintenQVD0

**Pull request merged**